src/components/BottomBar/BottomBarState.ts

```diff
@@ -66,6 +66,7 @@ export const addCourseToBottomBar = (course: FirestoreSemesterCourse): void => {
       vueForBottomBar.bottomCourseFocus = i;
       return;
     }
+    vueForBottomBar.isExpanded = true;
```

Comment:
The bottom bar won't expand if it returns early as above.

Reply:
Yes, +1 this ^^